docs/reference/server-config.md

```diff
@@ -359,6 +361,29 @@ Example:
 	</tr>
 </table>
 
+## Hot logging level reload
+We don't support to completely reload the whole configuration yet at present. What we support for now is:
+- increase logging level by signaling `SIGUSR1`
+- decrease logging level by signaling `SIGUSR2`
+
+Example:
+
+To increase logging level
+```
+$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```

Comment:
@HuKeping Thanks for checking!  I would be fine with documenting it as `kill -s SIGUSR1 PID` or `docker exec -i kill -s SIGUSR1 PID`, so long as we also document how to get the `PID` (e.g. `docker exec -i ps aux`) or something (and mention that it's not necessarily the PID 1 process).